src/services/refactors/inlineVariable.ts

```diff
@@ -152,7 +152,7 @@ function getInliningInfo(file: SourceFile, startPosition: number, tryWithReferen
     }
 
     // Try finding the declaration and nodes to replace via the reference token.
-    if (tryWithReferenceToken) {
+    if (tryWithReferenceToken || isShorthandPropertyAssignment(parent)) {

```

Comment:
Why is this condition modified? The purpose of `tryWithReferenceToken` is for a lightbulb to show up only in the declaration site, and to require explicit refactor requests in reference sites.

Reply:
Should _shorthand properties_ only be allowed to change with the `invoked` trigger reason?

Reply:
I believe so. Thoughts @DanielRosenwasser?

Reply:
That sounds right. The shorthand property is effectively considered a usage. So it should work similarly to the following case

```ts
let x = 123;
let y = {
  x: x/**/
};
```

Unless there's something you're noticing that we're missing @a-tarasyuk